### ISSUE TITLE
Keplr 2.0 wallet migration failing

### ISSUE THREAD

**Issue by Krewedk0** [chainapsis/keplr-wallet#738]
**Describe the bug**
Migration fails after some time with following error on 0.12.3 version:
```
Error: Coin type is not associated to chain
    at t.InExtensionMessageRequester.<anonymous> (678.bundle.js:1:223505)
    at Generator.next (<anonymous>)
    at a (678.bundle.js:1:222720)
```
There is some sort of fix around cointypes in 0.12.4 which could possibly resolve this issue, but 0.12.4 is not on chrome store for some reason.

**To Reproduce**
1. Open extension
2. Fill in password and press unlock
3. Wait for migration to fail with "Invalid password" error on the UI

**Comment by Thunnini** [comment]
Apologies, we are getting 0.12.4 reviewed in chrome webstore. It will be published tomorrow, lmk if this issue persists by tomorrow

**Comment by Krewedk0** [comment]
@Thunnini Just updated extension to 0.12.4 and getting the very same error in migration. Can't get past migration

**Comment by Thunnini** [comment]
This should be solved in v0.12.6

**Comment by Krewedk0** [comment]
Can confirm, problem solved in 0.12.6, closing the issue.
